apis/swagger.yml

```diff
@@ -3582,6 +3582,17 @@ definitions:
           items:
             type: "string"
 
+  CheckpointCreateOptions:
```

Comment:
We need to add more API side details for all these APIs.

like:
```
  /images/create:
    post:
      summary: "Create an image by pulling from a registry or importing from an existing source file"
      consumes:
```

Reply:
Already done, @allencloud